File: RxTools-library/src/main/java/com/vondear/rxtools/view/popupwindows/tools/RxCoordinates.java
package com.vondear.rxtools.view.popupwindows.tools;

import android.view.View;

public class RxCoordinates {

    int left;
    int top;
    int right;
    int bottom;

    public RxCoordinates(View view){
        int[] location = new int[2];
        view.getLocationOnScreen(location);
        left = location[0];
        right = left + view.getWidth();
        top = location[1];
        bottom = top + view.getHeight();
    }


}

File: RxUI/src/main/java/com/tamsiree/rxui/view/popupwindows/tools/RxCoordinates.kt
package com.tamsiree.rxui.view.popupwindows.tools

import android.view.View

/**
 * @author tamsiree
 */
class RxCoordinates(view: View) {
    var left: Int
    var top: Int
    var right: Int
    var bottom: Int

    init {
        val location = IntArray(2)
        view.getLocationOnScreen(location)
        left = location[0]
        right = left + view.width
        top = location[1]
        bottom = top + view.height
    }
}